document/core/exec/instructions.rst

```diff
@@ -464,6 +985,126 @@ Memory Instructions
    \end{array}
 
 
+.. _exec-load-extend:
+
+:math:`\V128\K{.}\LOAD{M}\K{x}L\_\sx~\memarg`
```

Comment:
Nit: can we use N instead of L here?

Reply:
`N` is already used later on, defined to be `M * L`. This follows the other memory instructions which use `N` as the number of bits actually loaded/stored.

Reply:
Yes, but N is an intermediate that does not actually occur in the formal rule, so it's a bit odd skipping it from that perspective.

Reply:
I see what you mean. I think we could simplify this, since `M*L` should always be 64, we can assert this (due to syntax) early on, and propagate 64 everywhere we use `N` or `M*L`.

Reply:
I would suggest simply inlining M\cdot L -- it is short enough and occurs only in two places I think.